ansys-grantami-serverapi-openapi/tests/conftest.py

```diff
@@ -32,7 +32,7 @@
     Generated by: https://github.com/swagger-api/swagger-codegen.git
 """
 
-import ansys.grantami.serverapi_openapi.models as models
+import ansys.grantami.serverapi_openapi.v252.models as models
```

Comment:
Is there a way we can run these tests on multiple versions of the generated code? Or I suppose, on the latest dev version? I can see us forgetting to change this and ending up with tests running against an old version of the package.

Ideally we'd import everything, and maybe xfail any specific versions where we know the tests don't work? Or import and run over a subset of all versions for each test if we know the behavior under test was only supported by a certain version. They should be open-ended though, to always pull in new tests, assuming we never break any existing functionality.